client/me/purchases/purchases-list/header/index.jsx

```diff
@@ -43,20 +51,29 @@ const PurchasesHeader = ( { section, translate } ) => {
 					{ translate( 'Payment Methods' ) }
 				</NavItem>
 
-				{ config.isEnabled( 'async-payments' ) && (
+				{ isEnabled( 'async-payments' ) && (
 					<NavItem path={ pendingPayments } selected={ section === 'pending' }>
 						{ translate( 'Pending Payments' ) }
 					</NavItem>
 				) }
 			</NavTabs>
+
+			{ section === 'billing' && (
+				<Search
+					pinned
+					fitsContainer
+					onSearch={ ( term ) => {
+						dispatch( setQuery( 'past', term ) );
+					} }
+					initialValue={ null }
+					placeholder={ translate( 'Search all receipts…' ) }
+					analyticsGroup="Billing"
+				/>
+			) }
 		</SectionNav>
 	);
-};
+}
 
 PurchasesHeader.propTypes = {
 	section: PropTypes.string.isRequired,
 };
-
-export default connect( ( state ) => ( {
```

Comment:
Nice.

Reply:
I think this might be why the team city test is failing.